### PR TITLE
Add permissions per group pane

### PR DESCRIPTION
**Pull request by magarnier**

### CODE REVIEW

File: languages/en/user-guide/project-admin.rst [hunk 1]
```diff
@@ -260,6 +260,19 @@ Those permissions concerns service who are in end of life:
 
    -  *Admin*: administrator of core SVN (single repository)
 
+Permissions per group
+`````````````````````
+This service enable you to check what can a user group do on your project.
```
Comment (nterray) [inline languages/en/user-guide/project-admin.rst]: Are you sure that this is a "service"?
s/enable/allows/
s/user group/users group/

File: languages/en/user-guide/project-admin.rst [hunk 2]
```diff
@@ -260,6 +260,19 @@ Those permissions concerns service who are in end of life:
...
+By default this service show you the global permissions defined for your project,
```
Comment (nterray) [inline languages/en/user-guide/project-admin.rst]: "service"?
[…] displays the global permissions defined for each service in the project

File: languages/en/user-guide/project-admin.rst [hunk 3]
```diff
@@ -260,6 +260,19 @@ Those permissions concerns service who are in end of life:
...
+if you want to check items permissions (like Git repositories, Packages, Releases ...)
+you will have to load all permissions.
```
Comment (nterray) [inline languages/en/user-guide/project-admin.rst]: […] to load them individually.

File: languages/en/user-guide/project-admin.rst [hunk 4]
```diff
@@ -260,6 +260,19 @@ Those permissions concerns service who are in end of life:
...
+
+You can use the filter to verify permissions for a specific user group.
```
Comment (nterray) [inline languages/en/user-guide/project-admin.rst]: s/user group/users group/

File: languages/en/user-guide/project-admin.rst (hunk 1 above)
Comment (magarnier) [reply languages/en/user-guide/project-admin.rst]: Done

**Pull request merged**